ansible/templates/vault.hcl.j2

```diff
@@ -0,0 +1,18 @@
+{% raw -%}
+
+ui = true
+disable_mlock = true
+
+api_addr      = "http://127.0.0.1:8200"
+cluster_addr  = "http://127.0.0.1:8201"
+
+storage "consul" {
+    address = "127.0.0.1:8500"
+    token = "b6e29626-e23d-98b4-e19f-c71a96fbdef7"
+}
+
+listener "tcp" {
+    address     = "127.0.0.1:8200"
+    tls_disable = 1
+}
```

Comment:
```suggestion
listener "tcp" {
  address = "{{ GetInterfaceIP \"docker0\" }}:8200"
}
listener "tcp" {
  address = "{{ GetInterfaceIP \"eth0\" }}:8200"
}
listener "tcp" {
  address = "{{ GetInterfaceIP \"eth1\" }}:8200"
}
```

Reply:
I am not sure that vault support such a configuration parsing
```
Error initializing listener of type tcp: listen tcp: lookup {{ GetInterfaceIP "docker0" }}: no such host
```